vic_run/src/runoff.c

```diff
@@ -39,48 +39,51 @@ runoff(cell_data_struct  *cell,
        soil_con_struct   *soil_con,
        double             ppt,
        double            *frost_fract,
-       int                dt,
+       double             dt,
```

Comment:
Yes, I check that the `runoff_steps_per_day` is a multiple of `model_steps_per_day` in `get_global_param()`.